examples/docker/basic_usage.py

```diff
@@ -0,0 +1,19 @@
+from pipeline import docker
+from pipeline import Pipeline, Variable, pipeline_function
+
+
+@pipeline_function
+def add_numbers(input_dict: dict) -> float:
+    return input_dict["in_1"] + input_dict["in_2"]
```

Comment:
Is dictionary input a requirement for using a `Pipeline` with `create_pipeline_api`, or is this just an example and one could use `add_number(x: int, y:int)` instead?

Reply:
I have actually changed the way args are sent now, so it was an issue with the inputs before. The full correction of this requires a change in the RunCreate schema to support generic inputs not just a str, which is what's now passed into the API. See [here](https://github.com/mystic-ai/pipeline-docker/blob/main/app/routes/run.py).

Reply:
I will correct this in this example to pass in ints instead, but it will not work with the current pipeline package.